agent/ecs_client/model/api/api-2.json

```diff
@@ -353,6 +429,21 @@
         {"shape":"InvalidParameterException"}
       ]
     },
+    "PutAccountSettingDefault":{
+      "name":"PutAccountSettingDefault",
+      "http":{
+        "method":"POST",
+        "requestUri":"/"
+      },
+      "input":{"shape":"PutAccountSettingDefaultRequest"},
+      "output":{"shape":"PutAccountSettingDefaultResponse"},
+      "errors":[
+        {"shape":"ServerException"},
+        {"shape":"ClientException"},
+        {"shape":"InvalidParameterException"}
+      ],
+      "internalonly":true
```

Comment:
i know it is automatically generated, but can you double check if we need to remove this?